src/App.js

```diff
@@ -10,66 +11,114 @@ import BrandAssetsPage from 'components/BrandAssetsPage';
 import CodebasePage from 'components/CodebasePage';
 import Manifesto from './components/Manifesto';
 import WorkerCompensation from './components/WorkerCompensation';
+import Blog from './components/Blog/Blog';
+import BlogPost from './components/Blog/BlogPost';
+
+const AppShell = styled.div`
+    width: 936px;
+    max-width: 100vw;
+    min-height: 100vh;
+
+    @media only screen and (max-width: 475px) {
+        ${(props) => (props.toggle ? 'position:fixed;' : '')};
+    }
+`;
 
 const App = () => {
+    const [mobileState, setMobileState] = useState(false);
+
     return (
         <HashRouter>
             <Switch>
                 <Route exact path="/">
                     <div className="landing-body-container">
-                        <div className="app-shell">
-                            <LandingPageHeader />
+                        <AppShell toggle={mobileState}>
+                            <LandingPageHeader
+                                setMobileState={setMobileState}
+                            />
                             <LandingPage />
                             <LandingPageFooter />
-                        </div>
+                        </AppShell>
```

Comment:
We shouldn't have any repeating components in our routes here. Instead, we should be able to wrap all the routes in the wrapping components we have since they repeat for each routes.

Reply:
yeah that needs to be reworked but it's not scope of this pr i also mentioned that to geronimo and john...